samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -60,6 +60,10 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
         console.log(`Call Id: ${callIdRef.current}`);
       }
     });
+    /* @conditional-compile-remove(call-transfer) */
+    adapter.on('transferRequested', (e) => {
```

Comment:
is this something we want to include in the sample app? or this an artifact from the test app you made?

Reply:
This is intended as transfer will now work in Calling sample app even though the Transfer page to indicate a transfer is not yet added.